src/sentry/static/sentry/app/views/settings/organizationIncidentRules/create.jsx

```diff
@@ -0,0 +1,1452 @@
+  [1563102000, []],
+  [1563103800, []],
+  [1563105600, []],
+  [1563107400, []],
+  [1563109200, []],
+  [1563111000, []],
+  [1563112800, []],
+  [1563114600, []],
+  [1563116400, []],
+  [1563118200, []],
+  [1563120000, []],
+  [1563121800, []],
+  [1563123600, []],
+  [1563125400, []],
+  [1563127200, []],
+  [1563129000, []],
+  [1563130800, []],
+  [1563132600, []],
+  [1563134400, []],
+  [1563136200, []],
+  [1563138000, []],
+  [1563139800, []],
+  [1563141600, []],
+  [1563143400, []],
+  [1563145200, []],
+  [1563147000, []],
+  [1563148800, []],
+  [1563150600, []],
+  [1563152400, []],
+  [1563154200, []],
+  [1563156000, []],
+  [1563157800, []],
+  [1563159600, []],
+  [1563161400, []],
+  [1563163200, []],
+  [1563165000, []],
+  [1563166800, []],
+  [1563168600, []],
+  [1563170400, []],
+  [1563172200, []],
+  [1563174000, []],
+  [1563175800, []],
+  [1563177600, []],
+  [1563179400, []],
+  [1563181200, []],
+  [1563183000, []],
+  [1563184800, []],
+  [1563186600, []],
+  [1563188400, []],
+  [1563190200, []],
+  [1563192000, []],
+  [1563193800, []],
+  [1563195600, []],
+  [1563197400, []],
+  [1563199200, []],
+  [1563201000, []],
+  [1563202800, []],
+  [1563204600, []],
+  [1563206400, []],
+  [1563208200, []],
+  [1563210000, []],
+  [1563211800, []],
+  [1563213600, []],
+  [1563215400, []],
+  [1563217200, []],
+  [1563219000, []],
+  [1563220800, []],
+  [1563222600, []],
+  [1563224400, []],
+  [1563226200, [{count: 30}]],
+  [1563228000, []],
+  [1563229800, []],
+  [1563231600, []],
+  [1563233400, []],
+  [1563235200, []],
+  [1563237000, []],
+  [1563238800, []],
+  [1563240600, []],
+  [1563242400, []],
+  [1563244200, []],
+  [1563246000, []],
+  [1563247800, []],
+  [1563249600, []],
+  [1563251400, []],
+  [1563253200, []],
+  [1563255000, []],
+  [1563256800, []],
+  [1563258600, []],
+  [1563260400, []],
+  [1563262200, []],
+  [1563264000, []],
+  [1563265800, []],
+  [1563267600, []],
+  [1563269400, []],
+  [1563271200, []],
+  [1563273000, []],
+  [1563274800, []],
+  [1563276600, []],
+  [1563278400, []],
+  [1563280200, []],
+  [1563282000, []],
+  [1563283800, []],
+  [1563285600, []],
+  [1563287400, []],
+  [1563289200, []],
+  [1563291000, []],
+  [1563292800, []],
+  [1563294600, []],
+  [1563296400, []],
+  [1563298200, []],
+  [1563300000, []],
+  [1563301800, []],
+  [1563303600, []],
+  [1563305400, []],
+  [1563307200, []],
+  [1563309000, []],
+  [1563310800, []],
+  [1563312600, []],
+  [1563314400, []],
+  [1563316200, []],
+];
+class IncidentRulesCreate extends React.Component {
+  state = {
+    width: null,
+  };
+
+  handleUpperBoundDrag = () => {};
+
+  handleLowerBoundDrag = () => {};
+
+  render() {
+    const {orgId} = this.props.params;
+
+    return (
+      <div>
+        <SettingsPageHeader title={t('New Incident Rule')} />
+        <LineChart
+          isGroupedByDate
+          forwardedRef={e => {
+            if (e && typeof e.getEchartsInstance === 'function') {
```

Comment:
What happens if you never get an echarts object?

Reply:
It should only happen in tests